master/internal/webhooks/postgres_webhook.go

```diff
@@ -0,0 +1,54 @@
+package webhooks
+
+import (
+	"context"
+
+	"github.com/uptrace/bun"
+
+	"github.com/determined-ai/determined/master/internal/db"
+)
+
+// AddWebhook adds a Webhook and its Triggers to the DB.
+func AddWebhook(ctx context.Context, w *Webhook) error {
+	return db.Bun().RunInTx(ctx, nil, func(ctx context.Context, tx bun.Tx) error {
+		_, err := tx.NewInsert().Model(w).Exec(ctx)
+		if err != nil {
+			return err
+		}
+		for _, t := range w.Triggers {
+			t.Webhookid = w.ID
+		}
+
+		if len(w.Triggers) != 0 {
+			_, err = tx.NewInsert().Model(&w.Triggers).Exec(ctx)
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	})
+}
+
+// GetWebhooks returns all Webhooks from the DB.
+func GetWebhooks(ctx context.Context) (Webhooks, error) {
+	webhooks := Webhooks{}
+	err := db.Bun().NewSelect().
+		Model(&webhooks).
+		Relation("Triggers").
+		Scan(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return webhooks, nil
+}
+
+// DeleteWebhook deletes a Webhook and its Triggers from the DB.
+func DeleteWebhook(ctx context.Context, id WebhookID) error {
+	return db.Bun().RunInTx(ctx, nil, func(ctx context.Context, tx bun.Tx) error {
+		_, err := db.Bun().NewDelete().Model((*Webhook)(nil)).Where("id = ?", id).Exec(ctx)
+		if err != nil {
+			return err
+		}
+		return nil
+	})
```

Comment:
there's still a redundant `RunInTx` here:

```suggestion
	_, err := db.Bun().NewDelete().Model((*Webhook)(nil)).Where("id = ?", id).Exec(ctx)
	if err != nil {
		return err
	}
	return nil
```

Reply:
Missed it before, updated now.